codeforboston/maple
# Card rework

# Summary

#1408

refactoring the basic Card component, per 1408; refactoring other components to come with follow up PRs

# Checklist

- N/A On the frontend, I've made my strings translate-able.
- [already present ] If I've added shared components, I've added a storybook story.
- N/A I've made pages responsive and look good on mobile.

# Known issues

the font sizes in bootstrap default to sizes that are a bit larger than our Figma; future PRs to change bootstrap's default font sizes

# Steps to test/reproduce

1. run storybook
2. open up the various subsections under Card and see that they are reasonable close to similar displays of Card on the live production server
